### PR TITLE
DRILL-6239: Add build and license badges to README.md

### PR DESCRIPTION
**Pull request by ilooner**
Add nice build and license badges that everyone else has. See a preview of what they look like here:

https://github.com/ilooner/drill/tree/DRILL-6239

### CODE REVIEW

**Comment by ilooner** [comment]
@arina-ielchiieva

**Comment by arina-ielchiieva** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Apache Drill
 
+[![Build Status](https://travis-ci.org/apache/incubator-openwhisk.svg?branch=master)](https://travis-ci.org/apache/drill)
```
Comment (vrozov) [inline README.md]: Please fix the link to point to drill (https://api.travis-ci.org/apache/drill.svg?branch=master). Consider adding maven central badge.

Comment (ilooner) [reply README.md]: Oh carp! Thanks for catching this.

**Comment by ilooner** [comment]
@vrozov Fixed bad link and added maven badge. Currently the maven badge points to drill's distribution pom. It's also interesting to note that only some of drill's artifacts are published on maven central. See https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.drill%22

**Pull request closed without merging**